frontend/src/liboperator/operator/devices.go

```diff
@@ -36,18 +40,13 @@ type Device struct {
 	clientCount int
 }
 
-type DeviceInfo struct {
-	DeviceId string `json:"device_id"`
-	GroupId  string `json:"group_id"`
-}
-
 type Group struct {
 	deviceIds []string
 }
 
 const DEFAULT_GROUP_ID = "default"
 
-func NewDevice(conn *JSONUnix, port int, info interface{}) *Device {
+func newDevice(id string, conn *JSONUnix, port int, privateData interface{}) *Device {
```

Comment:
suggest to use a new type here, i.e `type devicePrivateData interface{}` here.

```
func newDevice(id string, conn *JSONUnix, port int, data devicePrivateData) *Device

func groupIdFromPrivateData(data devicePrivateData) string
```

Reply:
ack, but will address it in a different PR since this one only changed the name of the field, the type was already there.